Packs/CommonScripts/Scripts/ParseWordDoc/ParseWordDoc.yml

```diff
@@ -15,7 +15,7 @@ args:
   default: true
   description: Docx file to parse
 scripttarget: 0
-dockerimage: demisto/docxpy:1.0.0.40261
+dockerimage: demisto/docxpy:1.0.0.89298
```

Comment:
try using office-utils instad and deprecate docxpy docker

Reply:
done